### ISSUE TITLE
Future support of template linking - outside of the JSX file.

### ISSUE THREAD

**Issue by Bengejd** [ionic-team/stencil#946]
[ ] bug report
[ ] feature request
[X] support request => Please do not submit support requests here, use one of these channels: https://stencil-worldwide.herokuapp.com/ or https://forum.ionicframework.com/

I was wondering if there was any support to decouple the template renderings into their own file. Similar to how things worked in all previous versions of Ionic before 4. I am not personally a fan of the JSX architecture and how it forces everything into one single file, which can lead to confusion in my opinion when working on a feature.

Basically, I'm just looking to see if there is any future planned support for `templateUrl: 'app.html',`syntax in Stencil.
